docs/README.md

```diff
@@ -1,503 +1,491 @@
+| `lineItems`                    | No            | ApplePayJS.ApplePayLineItem\[\];       | A set of line items that explain recurring payments and/or additional charges.                                                                                                                    |
+| `billingContact`               | No            | ApplePayJS.ApplePayPaymentContact      | The billing contact information for the shopper.                                                                                                                                                  |
+| `requiredBillingContactFields` | No            | ApplePayJS.ApplePayContactField\[\];   | The billing information that you require from the shopper to process the transaction.                                                                                                             |
+| `shippingContact`              | No            | ApplePayJS.ApplePayPaymentContact      | Shipping contact information for the user.                                                                                                                                                        |
+| `shippingMethods`              | No            | ApplePayJS.ApplePayShippingMethod\[\]; | The list of shipping methods available for a payment request.                                                                                                                                     |
+| `shippingType`                 | No            | ApplePayJS.ApplePayShippingType;       | An optional value that indicates how purchased items are to be shipped.                                                                                                                           |
+| `supportedCountries`           | No            | string\[ \]                            | A list of two-letter ISO 3166 country codes, used to limit payments to cards from specific countries or regions.                                                                                  |
+| `applicationData`              | No            | string                                 | Optional user-defined data.                                                                                                                                                                       |
+<br>
 
-- Have you replaced your intent id and client secret?
+#### Billing object
 
-```js
-// Enter your Payment Intent secret keys here
-const intent_id = 'replace-with-your-intent-id';
-const client_secret = 'replace-with-your-client-secret';
-```
+| **Property**    | **Required?** | **Type** | **Description**         |
+| --------------- | ------------- | -------- | ----------------------- |
+| `email`         | No            | string   | Shopper’s email         |
+| `first_name`    | No            | string   | Shopper’s first name    |
```

Comment:
some fields are required in billing
https://www.airwallex.com/docs/api?v=2022-02-16#/Payment_Acceptance/Payment_Intents/_api_v1_pa_payment_intents__id__confirm/post
<img width="1078" alt="image" src="https://user-images.githubusercontent.com/68316896/164578461-7066dbc1-a516-478e-b936-0766d8135401.png">

Reply:
@shirly-chen-awx :  The Address object which is part of Billing object contains 
`city`        
`country_code`
`postcode`    
`state`       
`street`      

first_name, last_name are already part of the Billing object

Reply:
I mean the second column, first_name and last_name should be required for billing
<img width="556" alt="image" src="https://user-images.githubusercontent.com/68316896/165204065-225a10fc-d3f0-42d2-8d32-e03f298daa11.png">

Reply:
Thanks @shirly-chen-awx!  Should I make "Address" object 'Required' in the "Billing" object?

Reply:
just to clarify,  I have made first_name, last_name Required in the billing object.  In the Address object, I have made country_code, street, city Required.  

I think Address object in Billing is conditional but we can leave it as optional.

Reply:
Let's follow the api documentation.